t/08-autotest.t

```diff
@@ -210,8 +210,7 @@ stderr_like { autotest::run_all } qr/Snapshots are not supported/, 'run_all outp
 is($died, 0, 'run_args test should not die if there is no run_args');
 is($completed, 0, 'run_args test should not complete if there is no run_args');
 
-eval { autotest::loadtest("tests/run_args.pm", name => 'alt_name', run_args => {foo => 'bar'}); };
-like($@, qr/The run_args must be a sub-class of OpenQA::Test::RunArgs/, 'error message mentions RunArgs');
+throws_ok { autotest::loadtest("tests/run_args.pm", name => 'alt_name', run_args => {foo => 'bar'}) } qr/The run_args must be a sub-class of OpenQA::Test::RunArgs/, 'error message mentions RunArgs';
```

Comment:
These lines are a bit too long for my taste.
Note that we temporarily have disabled line wrapping for perltidy.
So when we enable it again those lines would be wrapped, I suppose.
With that I can live with it.